components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -193,6 +192,18 @@ - (void)applyStyle:(MDCItemBarStyle *)style {
   }
 }
 
+- (void)updateTitleLines {
+  _titleLabel.numberOfLines = _style.titleNumberOfLines;
+  if (_style.titleNumberOfLines == 1) {
+    _titleLabel.adjustsFontSizeToFitWidth = NO;
+    _titleLabel.minimumScaleFactor = 1;
+  } else {
+    _titleLabel.adjustsFontSizeToFitWidth = YES;
+    // 0.85 is based on 12sp/14sp guidelines for single- or double-line text
+    _titleLabel.minimumScaleFactor = (CGFloat)0.85;
```

Comment:
As we only need this value if adjustFontSizeToFitWidth == YES.
Consider setting this value only once, on init.

Reply:
Done